src/pages/account-menu/in-a-drawer/Drawer.tsx

```diff
@@ -0,0 +1,178 @@
+import React, { useState } from 'react';
+import { Avatar, IconButton, Typography, makeStyles, Theme, useTheme } from '@material-ui/core';
+import { Dashboard, Notifications, ExitToApp, Settings, VpnKey } from '@material-ui/icons';
+import AssessmentIcon from '@material-ui/icons/Assessment';
+import LocationOnIcon from '@material-ui/icons/LocationOn';
+import {
+    Drawer,
+    DrawerBody,
+    DrawerHeader,
+    DrawerNavGroup,
+    DrawerLayout,
+    DrawerFooter,
+    NavItem,
+} from '@pxblue/react-components';
+import CloseIcon from '@material-ui/icons/Close';
+import WebAssetIcon from '@material-ui/icons/WebAsset';
+
+const backgroundImage = require('../../../assets/cubes_tile.png').default;
+const linearGradientOverlayImage = `linear-gradient(to right, rgba(0, 123, 193, 1) 22.4%, rgba(0, 123, 193, 0.2) 100%), url(${backgroundImage})`;
+
+const useStyles = makeStyles((theme: Theme) => ({
+    avatarSize: {
+        height: '48px',
+        width: '48px',
+    },
+    backgroundGradient: {
+        backgroundImage: `${linearGradientOverlayImage}`,
+        backgroundSize: 'contain',
+        backgroundPosition: 'right',
+    },
+    closeIcon: {
+        marginRight: `-${theme.spacing(2)}px`,
+        marginTop: `-${theme.spacing(4)}px`,
+    },
+    extendedHeader: {
+        width: '100%',
+        padding: '16px 16px 4px',
+    },
+    header: {
+        display: 'flex',
+        justifyContent: 'space-between',
+        zIndex: 1,
+        alignItems: 'center',
+        width: '100%',
+        height: '100%',
+    },
+    headerDetails: {
+        paddingTop: '16px',
+        position: 'relative',
+    },
+    subtitle: {
+        marginTop: `-${theme.spacing(0.5)}px`,
+    },
+}));
+
+type DrawerProps = {
+    open: boolean;
+    drawerToggler: () => void;
+};
+
+const avatarImage = require('../../../assets/avatar_40.png').default;
+
+export const PxbDrawer = (props: DrawerProps): JSX.Element => {
+    const { open, drawerToggler } = props;
+    const theme = useTheme();
+    const classes = useStyles(theme);
+    const variant = 'temporary';
+    const [selected, setSelected] = useState('1');
```

Comment:
I would be totally fine with not making any of the drawer selections functional...they're not the main focus of the example and they could be distracting. Just a thought.

Reply:
For showing below spec (active menu item's background shape should be round) I need to select menu item
![image](https://user-images.githubusercontent.com/10433274/129169317-7b05fc1b-1493-4ad2-bbf6-38e7ccac1126.png)